### ISSUE TITLE
Make "edit" command read stdin

### ISSUE THREAD

**Issue by sephii** [sephii/taxi#86]
It would be cool if the `edit` command could read stdin as a timesheet and merge it with the current timesheet. The user could then generate a timesheet from whatever source (eg. a calendar), pass it to `taxi edit` and then `taxi commit`.

**Comment by adri** [comment]
How did you imagine the merging to happen? I think in 90% of all cases its just appending `stdin` at the end of the current file.

**Comment by sephii** [comment]
Well it could work if you're sure you'll always use only this process to edit your timesheet. If you also edit your timesheet manually, and the date you want to add is already in your timesheet, you'd probably want to have the entries merged and the date to be only once in your timesheet. Also at the moment the ``edit`` command takes care of checking if you're using a top-down or a bottom-up notation in your timesheet and adds the new date to the right place (at the end or at the start of the file).

But yeah appending ``stdin`` could do the trick for simple cases, so I guess this feature can wait.

**Comment by adri** [comment]
When merging, do you mean replacing the whole day or merging single entries? I could imagine that to be hard to do automatically.

**Comment by sephii** [comment]
I was thinking about merging entries in a day, without dealing with duplicates and/or overlapping times, which would indeed be hard to do automatically.

**Comment by sephii** [comment]
Not needed for now.
